src/custom_nodes/east_ocr/config.json

```diff
@@ -0,0 +1,87 @@
+{
+    "model_config_list": [
+        {
+            "config": {
+                "name": "east",
+                "base_path": "/models/east_icdar2015_resnet_v1_50_rbox"
+            }
+        }
+    ],
+    "custom_node_library_config_list": [
+        {
+            "name": "east_image_extractor",
+            "base_path": "/libs/libcustom_node_east_ocr.so"
+        }
+    ],
+    "pipeline_config_list": [
+        {
+            "name": "detect_text_images",
+            "inputs": ["image"],
+            "nodes": [
+                {
+                    "name": "east_node",
+                    "model_name": "east",
+                    "type": "DL model",
+                    "inputs": [
+                        {"input_images": {
+                            "node_name": "request",
+                            "data_item": "image"}}
+                    ],
+                    "outputs": [
+                        {"data_item": "feature_fusion/Conv_7/Sigmoid",
+                            "alias": "scores"},
+                        {"data_item": "feature_fusion/concat_3",
+                            "alias": "geometry"}
+                    ]
+                },
+                {
+                    "name": "extract_node",
+                    "library_name": "east_image_extractor",
+                    "type": "custom",
+                    "params": {
+                        "original_image_width": "1920",
+                        "original_image_height": "1024",
+                        "target_image_width": "100",
+                        "target_image_height": "32",
+                        "convert_to_gray_scale": "true",
+                        "confidence_threshold": "0.9",
+                        "overlap_threshold": "0.3",
+                        "max_output_batch": "100",
+                        "debug": "true",
+                        "image_layout": "NCHW"
+                    },
+                    "inputs": [
+                        {"image": {
+                            "node_name": "request",
+                            "data_item": "image"}},
+                        {"scores": {
+                            "node_name": "east_node",
+                            "data_item": "scores"}},
+                        {"geometry": {
+                            "node_name": "east_node",
+                            "data_item": "geometry"}}
+                    ],
+                    "outputs": [
+                        {"data_item": "text_images",
+                            "alias": "text_images"},
+                        {"data_item": "text_coordinates",
+                            "alias": "text_coordinates"},
+                        {"data_item": "confidence_levels",
+                            "alias": "confidence_levels"}
+                    ]
```

Comment:
Missing CRNN node?

Reply:
yes, that will be added later by a PR from @dkalinowski

Reply:
After @atobiszei is done with review I will add it to this MR.

Reply:
Done